governance/understanding.rst

```diff
@@ -234,13 +233,7 @@ roles.
 Voting on proposals is updated in real time via P2P messages and stored
 by Dash in cache files, so current winning proposals and the total
 allocation of the available budget are always open and visible to
-everyone. `Dash Nexus <https://app.dashnexus.org/>`__ is a popular site used
```

Comment:
Let's switch this to link to https://www.dashcentral.org/budget instead of dropping the info altogether.